lib/_http_common.js

```diff
@@ -43,6 +43,16 @@ var kOnHeadersComplete = HTTPParser.kOnHeadersComplete | 0;
 var kOnBody = HTTPParser.kOnBody | 0;
 var kOnMessageComplete = HTTPParser.kOnMessageComplete | 0;
 
+var kHeaders = HTTPParser.kHeaders | 0;
+var kUrl = HTTPParser.kUrl | 0;
+var kVersionMajor = HTTPParser.kVersionMajor | 0;
+var kVersionMinor = HTTPParser.kVersionMinor | 0;
+var kMethod = HTTPParser.kMethod | 0;
+var kStatusCode = HTTPParser.kStatusCode | 0;
+var kStatusMessage = HTTPParser.kStatusMessage | 0;
+var kUpgrade = HTTPParser.kUpgrade | 0;
+var kShouldKeepAlive = HTTPParser.kShouldKeepAlive | 0;
```

Comment:
I haven't actually tried but now that the module is strict mode code, changing var to const should make it possible for V8 to inline the value instead of retrieving it from the closure context.  That's the theory, anyway; in practice, the optimizing compiler is still a bit finicky in the presence of let/const-scoped variables.  You'd have to check with --trace_opt/--trace_deopt.